vmdb/app/models/orchestration_stack_openstack_infra.rb

```diff
@@ -24,4 +24,9 @@ def raw_update_stack(options)
       connection.update_stack(stack, planning_data)
     end
   end
+
+  def update_ready?(state)
+    # Update is possible only when in complete or failed state, otherwise API returns exception
+    state.end_with?("_COMPLETE", "_FAILED")
```

Comment:
I guess the logging would be done on another level. E.g. in automate script or UI side. At least that is how I wrote the automate script.